### ISSUE TITLE
HiC-Pro version error message even though I have HiC-Pro installed

### ISSUE THREAD

**Issue by margaretwoodhouse** [ay-lab/FitHiChIP#43]
Hi there:

I'm running the most recent version of FitHiChip that I git cloned to my computer. I'm running FitHiChip through the singularity. I have all the necessary programs installed, including HiC-Pro version 2.11.4. However, whenever I run FitHiChip, I keep getting the error

```
HiC-pro is installed in the system
ERROR ====>>> HiC-pro should have version >= 2.11.4 !!! FitHiChIP quits !!!
```

My config file has the correct path to HiC-Pro, so I don't know why it's giving me this error message when it should be working. 

Thanks!

**Comment by ay-lab** [comment]
Hi @margaretwoodhouse 
Thanks for your query. Are you running "FitHiChIP_Singularity.sh" ? In such a case, please try to run the script "FitHiChIP_HiCPro.sh" directly. The singularity file needs to be updated, which I'll do soon. 
You can check the sample script file "sample_script.sh", and choose any of the configuration files (like configfile_BiasCorrection_CoverageBias) to run.
If that does not solve your problem, please let me know.

**Comment by margaretwoodhouse** [comment]
Hi, there:

Thanks for your quick response.

I ran FitHiChIP_HiCPro.sh directly, as suggested, but got the following error:

```
no HiC-Pro in (/usr/lib64/qt-3.3/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/mw/.local/bin:/home/mw/bin)
ERROR ===>>>> HiC-pro is not installed in the system - FitHiChIP quits !!!
```

It is looking for HiC-Pro in a path different from the one I gave it. I'm running these programs on an HPC and it looks to be only searching in the root directories instead of my local project paths despite the commands I give it in the config file. It seems able to read all the files from the paths I gave it in the config file; but for whatever reason, it's ignoring the path command I gave it for HiC-Pro. I even made a symbolic link from /home/mw2/.local/bin to the project directory path that contains HiC-Pro, and I'm still getting the above error. How do I get around this?

Thanks!
